lib/omniauth/strategies/facebook.rb

```diff
@@ -9,18 +9,15 @@ module Strategies
     class Facebook < OmniAuth::Strategies::OAuth2
       class NoAuthorizationCodeError < StandardError; end
 
-      DEFAULT_SCOPE = 'email'
+      DEFAULT_SCOPE = 'email'.freeze
+      API_VERSION = 'v2.6'.freeze
 
       option :client_options, {
-        :site => 'https://graph.facebook.com',
+        :site => "https://graph.facebook.com/#{API_VERSION}",
         :authorize_url => "https://www.facebook.com/dialog/oauth",
```

Comment:
https://developers.facebook.com/docs/facebook-login/manually-build-a-login-flow doesn't use version. See `Invoking the login dialog` section